stata_kernel/stata_session.py

```diff
@@ -299,7 +299,7 @@ def expect(self, text, child, md5, text_to_exclude=None, display=True):
         error_re = r'^r\((\d+)\);'
 
         # The minimum linesize in Stata is 40 characters
-        g_exp = r'\(file {}'.format(self.cache_dir_str[:34])
+        g_exp = r'\(?file {}'.format(self.cache_dir_str[:34])
```

Comment:
Probably good to add a comment above this noting that the `(` doesn't exist in Stata 17

Reply:
Thanks for the feedback @kylebarron , I've added a comment in [97709b6](https://github.com/kylebarron/stata_kernel/pull/394/commits/97709b6c46c1a01c670283ebd05920270044187d)